code/modules/surgery/organs/organ_external.dm

```diff
@@ -138,6 +138,28 @@
 	replaced(H)
 	return 1
 
+/obj/item/organ/external/attack(mob/living/carbon/C, mob/user)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(H.species && H.species.name == "Machine" && status == ORGAN_ROBOT)
```

Comment:
❄️

Reply:
As is the Para IPC way, yes. Let me know when you guys get species traits so I can just use `EASYLIMBATTACHMENT`. Until then this is the only way I know of to accomplish this.

Reply:
Which kinda begs the question: your statement seems to indicate you believe we'd favor a solution such based on the traits (we do). If you knew that, why not implement that system as opposed to doing something that would likely (and as a matter of fact did) get labeled as "snowflake".